### DISCUSSION TITLE
Any update for Positron: the new R IDE

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#69] [App, Extra, Book suggestions]
I am writing to inquire about the potential addition of Positron, the new R IDE, to the DAT Linux OS repository or package manager. As a user working extensively with R for data science and analytics, I am excited about the features and improvements that Positron offers for R programming and would greatly appreciate having access to it directly through DAT Linux OS.

**Comment by oss-linux-etc** [comment]
This is on the radar. Positron is still in beta, and some features are unstable or unfinished.
